IaC/rel.cd/JenkinsStack.yml

```
@@ -712,28 +706,33 @@ Resources:
                   wget -O /mnt/$JENKINS_HOST/nginx-google-oauth.lua \
                     https://raw.githubusercontent.com/cloudflare/nginx-google-oauth/master/access.lua
 
-                  chkconfig openresty on
-                  service openresty start
+                  systemctl enable --now openresty
               }
 
               setup_letsencrypt() {
                   install -d /usr/share/nginx/html
-                  certbot --debug --non-interactive certonly --agree-tos --register-unsafely-without-email --webroot -w /usr/share/nginx/html --keep -d $JENKINS_HOST
+                  if [[ -d /mnt/ssl_backup ]]; then
+                    rm -rf /etc/letsencrypt
+                    rsync -aHSv /mnt/ssl_backup/ /etc/letsencrypt/
```

JFYI `rm` not needed if `--delete` specified 
```suggestion
                    rsync -aHSv --delete /mnt/ssl_backup/ /etc/letsencrypt/
```

---

Fixed